react/LocationSearch.tsx

```diff
@@ -1,50 +1,92 @@
 import React, { useState } from 'react'
-import { Input, IconSearch, IconClear } from 'vtex.styleguide'
 import { FormattedMessage } from 'react-intl'
+import { Input, IconSearch, IconClear } from 'vtex.styleguide'
+import {
+  Combobox,
+  ComboboxInput,
+  ComboboxOption,
+  ComboboxPopover,
+  ComboboxList,
+} from '@reach/combobox'
+import '@reach/combobox/styles.css'
 
+import { addresses as mockedAddresses } from './addresses'
 import styles from './LocationSearch.css'
 
-const LocationSearch: React.FC = () => {
-  const [inputValue, setInputValue] = useState<string>('')
+const useAddressMatch = (term: string): string[] => {
```

Comment:
this isn't a hook, is it?

Reply:
It isn't a hook, but now I guess it should be an effect hook, right?

Reply:
hmm not necessarily... maybe a memo, but I don't think it would make much of a difference here. might be better to define it as a plain function and if we run into trouble later we could convert it to a hook

Reply:
That means literally a function like `getAddresses` and run it when the input changes?